### PR TITLE
Enable deployment of py-torch@1.7.0

### PR DESCRIPTION
**Pull request by pramodk**

### CODE REVIEW

File: deploy/environments/libraries.yaml [hunk 1]
```diff
@@ -158,6 +159,7 @@ spack:
     - py-sphinx-rtd-theme
     - py-sympy
     - py-tensorflow@2.4.2
+    - py-torch@1.7.0 cuda_arch=70
```
Comment (matz-e) [inline deploy/environments/libraries.yaml:162]: Same failure mode as in ML's PR… can't see the compiler on BlueBrain5…

**Comment by matz-e** [comment]
Seems like we're doing the same work in #1295, and see the same failures. Maybe we can continue this work there?

**Comment by pramodk** [comment]
Ok, need some additional time to come to this....

**Comment by matz-e** [comment]
I'm going to close this. The next deployment is at 1.10.0 already!

**Pull request closed without merging**